IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcImprovedGround/DocEntity.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcReinforcedSoil" UniqueId="92c17199-1655-4d3b-b443-7f88b06fcbce" BaseDefinition="IfcEarthworksElement" EntityFlags="32">
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcImprovedGround" UniqueId="92c17199-1655-4d3b-b443-7f88b06fcbce" BaseDefinition="IfcEarthworksElement" EntityFlags="32">
 	<Attributes>
-		<DocAttribute Name="PredefinedType" UniqueId="f6349eb8-a26a-4366-9ac1-5e81aa2a288b" DefinedType="IfcReinforcedSoilTypeEnum" AttributeFlags="1">
-			<Documentation>Identifies the predefined type of a reinforced soil elements. This type may associate additional specific property sets.</Documentation>
+		<DocAttribute Name="PredefinedType" UniqueId="f6349eb8-a26a-4366-9ac1-5e81aa2a288b" DefinedType="IfcImprovedGroundTypeEnum" AttributeFlags="1">
+			<Documentation>Identifies the predefined type (methods) of improving the ground. This type may associate additional specific property sets.</Documentation>
 		</DocAttribute>
 	</Attributes>
 	<WhereRules>
 		<DocWhereRule Name="CorrectPredefinedType" UniqueId="e533e402-9d57-4728-be40-35448bd6e3d8">
 			<Documentation>Either the _PredefinedType_ attribute is unset, or the inherited attribute _ObjectType_ shall be provided, if the _PredefinedType_ is set to USERDEFINED.</Documentation>
 			<Expression>NOT(EXISTS(PredefinedType)) OR
```

Comment:
Looks like changing to HasObjectType again and making PredefinedType mandatory since on IfcObjectType is defined.

Reply:
Should be fixed now.